src/fragments/30/fragment30.c

```diff
@@ -1,3 +1,7 @@
-#include "global.h"
+#include "fragment30.h"
 
-#pragma GLOBAL_ASM("asm/us/nonmatchings/fragments/30/fragment30/func_88A00020.s")
+// unused wrapper?
+s32 func_88A00020(UNUSED s32 arg0, UNUSED s32 arg1) {
+    func_80029048((( (u32) &D_88000000 & 0x0FF00000) >> 0x14) - 0x10, fragment22_ROM_START, fragment23_ROM_START, 1, 0);
```

Comment:
This is the FRAGMENT_LOAD_AND_CALL2 macro from here: https://github.com/pret/pokestadium/blob/d218ac936121f7fb3f7b852c8a22c450460b0cb8/include/macros.h#L67-L69, use that instead if you can. You jsut need to pass `fragment22` to it and it'll calculate the first 2 args for you.